README.md

```diff
@@ -18,70 +19,75 @@ so using the Crossplane CLI in a Kubernetes cluster where Crossplane is
 installed:
 
 ```console
-kubectl crossplane install provider crossplanecontrib/provider-helm:master
+crossplane xpkg install provider xpkg.upbound.io/crossplane-contrib/provider-helm:v0.17.0
 ```
 
-You may also manually install `provider-helm` by creating a `Provider` directly:
+Then you will need to create a `ProviderConfig` that specifies the credentials
+to connect to the Kubernetes API. This is commonly done within a `Composition`
+by storing a `kubeconfig` into a secret that the `ProviderConfig` references. An
+example of this approach can be found in
+[`configuration-aws-eks`](https://github.com/upbound/configuration-aws-eks/blob/release-0.7/apis/composition.yaml#L427-L452).
 
-```yaml
-apiVersion: pkg.crossplane.io/v1
-kind: Provider
-metadata:
-  name: provider-helm
-spec:
-  package: "crossplanecontrib/provider-helm:master"
-```
-
-## Design 
+### Quick start
 
-See [the design document](https://github.com/crossplane/crossplane/blob/master/design/one-pager-helm-provider.md).
+An alternative, that will get you started quickly, is to reuse existing
+credentials from within the control plane.
 
-## Developing locally
-
-Start a local development environment with Kind where `crossplane` is installed:
+First install `provider-helm` with [additional
+configuration](./examples/provider-config/provider-incluster.yaml) to bind its
+service account to an existing role in the cluster:
 
+```console 
+kubectl apply -f ./examples/provider-config/provider-incluster.yaml
 ```
-make local-dev
+
+Then simply create a
+[`ProviderConfig`](./examples/provider-config/provider-config-incluster.yaml)
+that uses an `InjectedIdentity` source:
+  
+```console 
+kubectl apply -f ./examples/provider-config/provider-config-incluster.yaml
 ```
 
-Run controller against the cluster:
+`provider-helm` will then be installed and ready to use within the cluster. You
+can now create `Release` resources, such as [sample
+release.yaml](examples/sample/release.yaml).
 
-```
-make run
+```console
+kubectl create -f examples/sample/release.yaml
 ```
 
-Since controller is running outside of the Kind cluster, you need to make api server accessible (on a separate terminal):
+## Design 
 
-```
-sudo kubectl proxy --port=8081
-```
+See [the design
+document](https://github.com/crossplane/crossplane/blob/master/design/one-pager-helm-provider.md).
 
-### Testing in Local Cluster
+## Developing locally
 
-1. Prepare provider config for local cluster:
-    1. If helm provider running in cluster (e.g. provider installed with crossplane):
-    
-        ```
-        SA=$(kubectl -n crossplane-system get sa -o name | grep provider-helm | sed -e 's|serviceaccount\/|crossplane-system:|g')
-        kubectl create clusterrolebinding provider-helm-admin-binding --clusterrole cluster-admin --serviceaccount="${SA}"
-        kubectl apply -f examples/provider-config/provider-config-incluster.yaml
```

Comment:
I dont think this scenario has much value now given the quick start option above that creates the cluster role binding in a more crossplane native way 😉 e.g. https://github.com/crossplane-contrib/provider-helm/blob/master/examples/provider-config/provider-incluster.yaml